### DISCUSSION TITLE
v5.0.1_39

### DISCUSSION THREAD

**Discussion by Brog33** [Alexco500/sidplay5#19] [General]
Hey, just wanna say great work with the update :) 
I can't seem to find the SID model switcher, though.

**Comment by Alexco500** [comment]
Thanks.
It is the small "chip" icon on the right, direct below the playlists.
I would like to change that "chip" icon later to some SID thing, but I am no designer :-)

**Comment by Brog33** [comment]
aha! thanks for telling me. here's why i didn't find it .. it's not responsive, it was hidden in the overflow :)
to me it would make more sense to place the sid switcher together with the UI elements in the upper part of the window. e.g. next to the sub tune selector. 
 
![image](https://user-images.githubusercontent.com/1303537/235319372-7b7c9f92-f2a2-4e5f-9c08-28471790ae8d.png)

**Comment by Alexco500** [comment]
Oh, ok, that needs to be fixed. 
Putting it into the song info box could be difficult, since this is an OpenGL view and Apple does not really endorse this any more.
Maybe I fix the current idea and we think about that then.

**Reply by Brog33** [reply]
While we're on the topic, that cog icon in the bottom left - instinctively i assume it will open the settings window. 
Just an idea: Perhaps it could be removed, and instead add a drop-down to the plus icon (similar to the SID toggle) where the user can choose between "new playlist" and "new smart playlist". 🤔
